packages/headless/src/app/commerce-engine/commerce-engine-configuration.ts

```diff
@@ -58,7 +55,11 @@ export const commerceEngineConfigurationSchema =
             '@coveo/quantic': nonEmptyString,
           },
         }),
-        trackingId: requiredNonEmptyString,
+        trackingId: new StringValue({
+          required: true,
+          emptyAllowed: false,
+          regex: /^[a-zA-Z0-9_\-\\.]{1,100}$/,
```

Comment:
I think this regex accepts `\` when it shouldn't. I think this is what we want:
```suggestion
          regex: /^[a-zA-Z0-9_\-.]{1,100}$/,
```

https://regex101.com/r/oW0QSQ/2

Reply:
good catch